wgpu-core/src/command/compute.rs

```diff
@@ -309,8 +309,10 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
         let (device_guard, mut token) = hub.devices.read(&mut token);
 
         let (mut cmd_buf_guard, mut token) = hub.command_buffers.write(&mut token);
-        let cmd_buf = CommandBuffer::get_encoder_mut(&mut *cmd_buf_guard, encoder_id)
-            .map_pass_err(init_scope)?;
+        let result = CommandBuffer::get_encoder_mut(&mut *cmd_buf_guard, encoder_id);
+        // Call using fully qualified syntax, to placate rust-analyzer.
+        // https://github.com/rust-lang/rust-analyzer/issues/12247
+        let cmd_buf = MapPassErr::<&mut CommandBuffer<A>, _>::map_pass_err(result, init_scope)?;
```

Comment:
How about?

I've actually done this for a lot of other things in my tracker refactor, as I was touching all that code

```suggestion
        let cmd_buf: &mut CommandBuffer<A> = CommandBuffer::get_encoder_mut(&mut *cmd_buf_guard, encoder_id)
```

Reply:
That works too.